user-access-internal-model/src/main/java/no/unit/nva/useraccessmanagement/internals/UserScanResult.java

```diff
@@ -0,0 +1,36 @@
+package no.unit.nva.useraccessmanagement.internals;
```

Comment:
class was moved from the EventBasedScanHandler to the internals modules and was made a separate class